src/main/java/com/cloudbees/jenkins/plugins/sshagent/mina/MinaRemoteAgentStarter.java

```diff
@@ -24,15 +24,23 @@
 
 package com.cloudbees.jenkins.plugins.sshagent.mina;
 
+import jenkins.security.MasterToSlaveCallable;
+
 import com.cloudbees.jenkins.plugins.sshagent.RemoteAgent;
+
 import hudson.model.TaskListener;
-import hudson.remoting.Callable;
 import hudson.remoting.Channel;
 
 /**
  * Callable to start the remote agent.
  */
-public class MinaRemoteAgentStarter implements Callable<RemoteAgent, Throwable> {
+public class MinaRemoteAgentStarter extends MasterToSlaveCallable<RemoteAgent, Throwable> {
+
+    /**
+     * 
+     */
+    private static final long serialVersionUID = 2909967505958103050L;
```

Comment:
Aha! OK, with Java 1.6, 1.7 and 1.8 using the 1.7 release of the plugin I have:

```
$ serialver -classpath /Users/stephenc/.m2/repository/org/jenkins-ci/plugins/ssh-agent/1.7/ssh-agent-1.7.jar:/Users/stephenc/.m2/repository/org/jenkins-ci/main/remoting/2.33/remoting-2.33.jar:/Users/stephenc/.m2/repository/org/jenkins-ci/main/jenkins-core/1.532.2/jenkins-core-1.532.2.jar  com.cloudbees.jenkins.plugins.sshagent.mina.MinaRemoteAgentStarter
com.cloudbees.jenkins.plugins.sshagent.mina.MinaRemoteAgentStarter:    private static final long serialVersionUID = -3757105406876098311L;
```

So `private static final long serialVersionUID = -3757105406876098311L` is what we should have. Anything else is a :bug: